### PR TITLE
https update 

### PR DESCRIPTION
**Pull request by ahmadfarkhanmn**
due to https set up, new update

### CODE REVIEW

**Comment by ahmadfarkhanmn** [comment]
due to https setup

**Review by ahmadfarkhanmn — COMMENTED** [review]
update

File: _data/homepage.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ agency-logo: /images/Title FA-01.png
 
 # Hero Banner 
 fontsize: 200pt
-hero-banner: /MTLS_UnderConstruction.jpg
+hero-banner: /images/MTLS_UnderConstruction.jpg
```
Comment (ahmadfarkhanmn) [inline _data/homepage.yml]: please remove /images

File: _data/homepage.yml [hunk 2]
```diff
@@ -23,8 +23,17 @@ hero-banner: /MTLS_UnderConstruction.jpg
 #Google Analytics
 google-analytics: UA-124401955-1
 
+key-highlights:
 
-
```
Comment (ahmadfarkhanmn) [inline _data/homepage.yml]: do we have to remove the area

**Review by limxuanhan — APPROVED** [review]

**Pull request merged**